content/glossary/drought-scenario.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Drought Scenarios
+---
+
+Two drought scenarios were constructed for California’s Fourth Climate Change Assessment to investigate implications of an extended dry spell:
+
+- Late 21st Century Drought: This scenario represents a late century dry spell from 2051–2070 identified from 
+    the HadGEM2-ES RCP 8.5 simulation. The extended drought scenario is based on the average 
+    annual precipitation over 20 years. This average value equates to 78% of historical median 
+    annual precipitation averaged over the North Coast and Sierra California Climate Tracker regions..
```

Comment:
remove extra period

Reply:
fixed in  https://github.com/berkeley-gif/caladapt-website-2021/pull/125/commits/6e747fb64773a30dd1999a9464f1edd8ab09ccf9